common/src/java/com/zimbra/common/account/ZAttrProvisioning.java

```diff
@@ -70,6 +70,22 @@ public static AccountStatus fromString(String s) throws ServiceException {
         public boolean isPending() { return this == pending;}
     }
 
+    public static enum AddressListStatus {
+        active("active"),
+        inactive("inactive");
+        private String mValue;
+        private AddressListStatus(String value) { mValue = value; }
+        public String toString() { return mValue; }
+        public static AddressListStatus fromString(String s) throws ServiceException {
+            for (AddressListStatus value : values()) {
+                if (value.mValue.equals(s)) return value;
+             }
+             throw ServiceException.INVALID_REQUEST("invalid value: "+s+", valid values: "+ Arrays.asList(values()), null);
+        }
+        public boolean isActive() { return this == active;}
+        public boolean isInactive() { return this == inactive;}
```

Comment:
is isInactive() really needed? there is isActive() just above isInactive() which can be used to get whether adress list is active or not.

Reply:
these methods are auto generated by the generate-getters ant target.